package.json

```diff
@@ -33,7 +30,7 @@
     "start": "node lib/server",
     "watch": "babel scripts/watch.js | node",
     "test": "npm run lint && npm run check && npm run test:only",
-    "test:only": "mocha $npm_package_options_mocha",
+    "test:only": "jest",
```

Comment:
ideally, I would like to name it `test` but to avoid pilling up too many changes in 1 PR. I've decided to leave if backward compatible.